src/server/handle-redirects.js

```diff
@@ -31,6 +31,8 @@ export default ({ server, config }) => {
   if (fs.existsSync(redirectsFile)) {
     const redirectsFromFile = JSON.parse(fs.readFileSync(redirectsFile, 'utf8'))
     setRedirects(server, Object.assign({}, redirects, redirectsFromFile))
```

Comment:
Out of interest: Does the Object.assign not work merging the redirects from config and redirects from file here? Have you found this not to work?

Reply:
It only gets into this block if `redirectsFile` exists, so `setRedirects` will never get called if `root/redirects.json` doesn't exist.

Reply:
Doi - good spot.